drivers/wifi/nrf700x/Kconfig

```diff
@@ -62,12 +62,22 @@ config NRF700X_STA_MODE
 
 config NRF700X_AP_MODE
 	bool "Enable Access point mode"
+	depends on NRF700X_STA_MODE
```

Comment:
Can we remove this dependency, so that AP mode can be compiled independent of STA mode, providing an opportunity to optimize memory by excluding STA specific functionality?

Reply:
This is not related to raw TX mode dependencies, right? 

Makes sense to me to have an option for AP-only support - to minimize footprint.

Reply:
> This is not related to raw TX mode dependencies, right?

No, please see the commit log.

Reply:
> Can we remove this dependency, so that AP mode can be compiled independent of STA mode, providing an opportunity to optimize memory by excluding STA specific functionality?

This needs a wider review, so, dropped that commit.